crates/topos-tce-storage/src/validator/mod.rs

```diff
@@ -118,6 +118,23 @@ impl ValidatorStore {
         Ok(self.pending_tables.pending_pool.iter()?.collect())
     }
 
+    pub fn get_next_pending_certificates(
+        &self,
+        from: &PendingCertificateId,
+        number: usize,
+    ) -> Result<Vec<(PendingCertificateId, Certificate)>, StorageError> {
+        debug!(
+            "Get next pending certificates from {} (max: {})",
+            from, number
+        );
+        Ok(self
+            .pending_tables
+            .pending_pool
+            .iter_at(from)?
```

Comment:
Not a task for this PR ofc but we should have a think about building a good benchmark for code like this so we know what performance characteristics we can expect for different DB sizes. Rocksdb iterators are quirky and sometimes surprising.

Reply:
Indeed, we should have a long term prioritization of those underlying subject that doesn't really fit in features.

Reply:
No strong opinion on how to capture these things, as long as they are written down somewhere in a ticket with a proper label.